docs/Catalog/VisualizationDashboards.md

```diff
@@ -350,3 +352,22 @@ All filters and columns will be restored:
   }
 ]
 ```
+
+## Igv
+
+To render an [Igv](https://igv.org/), you provide a JSON file (a dictionary that
+specifies the Igv [browser configuration options](https://github.com/igvteam/igv.js/wiki/Browser-Creation#browser-configuration-options))
+and you set the `"types"` property to `[ "igv" ]`.
+
+Alternatively, you can browse that JSON file in Quilt web and switch viewer from JSON to Igv
+
+```json
+// quilt_summarize.json
+[
+  {
+    "path": "igv-options-file.json",
+    "title": "Awesome genome",
+    "types": ["igv"]
+  }
+]
+```
```

Comment:
It's impossible to make a suggestion with three backticks in GH. Anyway, please add this sentence on the next line:
```
In the above example, `igv-options-file.json` is an
[IGV browser configuration](https://github.com/igvteam/igv.js/wiki/Browser-Creation#browser-configuration-options).
```